sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/Generated/Models/App.cs

```diff
@@ -21,7 +22,7 @@ namespace Microsoft.Azure.Management.IotCentral.Models
     /// The IoT Central application.
     /// </summary>
     [Rest.Serialization.JsonTransformation]
-    public partial class App : Resource
+    public partial class App : IResource
```

Comment:
@Yao725, this App class is missing the location and tags fields defined in the specification. In the specification, the App definition is defined to inherit all properties of the TrackedResource definition, which in turn inherits from Resource. Do you have any ideas why these are missing?

Reply:
Will take a look

Reply:
@Yao725, for this one it looks like the problem is caused by the fact that the App definition in the swagger spec is decorated with the "x-ms-azure-resource" property. When I remove this, the App class properly inherits from TrackedResource.

Reply:
I agree with your comment, on the one hand, you marked `App` as `azure resource`, but on the other hand, the common type `Resource` which is inherited by `TrackedResource`, is also marked as `azure resource`. I think it is this duplication of definitions that causes this error. After I removed this one in `App`, the generated code is right:
```csharp
public partial class App : TrackedResource
```

Reply:
I'll remove that property from the spec.

Reply:
PR for this update to the spec: https://github.com/Azure/azure-rest-api-specs/pull/18736